### PR TITLE
Add card submitting functionality

### PR DESCRIPTION
**Pull request by KubaZm**

### CODE REVIEW

File: frontend/src/lib/components/TableDisplayer/TableDisplayer.svelte [hunk 1]
```diff
@@ -22,6 +26,20 @@
...
     };
 
+    const submitCard = () => {
+        selectCard($usernameStore, tableId, selectedCard).then(() => {
+            disabled = true;
+            submitted = true;
+        }).catch(errorMessage => toastComponent.toast(errorMessage));
+    }
+
+    function onClickCard(card) {
```
Comment (Zuretto) [inline frontend/src/lib/components/TableDisplayer/TableDisplayer.svelte]: please use arrow function instead of `function`

**Review by kezc — DISMISSED** [review]
LGTM

File: frontend/src/lib/components/TableDisplayer/TableDisplayer.svelte (hunk 1 above)
Comment (Zuretto) [reply frontend/src/lib/components/TableDisplayer/TableDisplayer.svelte]: and add types

**Review by Zuretto — CHANGES_REQUESTED** [review]

**Review by kezc — APPROVED** [review]
LGTM

**Review by Zuretto — APPROVED** [review]

**Pull request merged**